Makefile

```diff
@@ -1,11 +1,16 @@
 SOURCE_FILES := $(shell test -e src/ && find src -type f)
+VERSION := $(shell sed --posix -n 's,^version = \"\(.*\)\",\1,p' Cargo.toml)
 
 policy.wasm: $(SOURCE_FILES) Cargo.*
 	cargo build --target=wasm32-wasi --release
 	cp target/wasm32-wasi/release/*.wasm policy.wasm
 
-annotated-policy.wasm: policy.wasm metadata.yml
-	kwctl annotate -m metadata.yml -o annotated-policy.wasm policy.wasm
+artifacthub-pkg.yml: metadata.yml Cargo.toml
+	kwctl scaffold artifacthub --metadata-path metadata.yml --version $(VERSION) \
+		--questions-path questions-ui.yml --output artifacthub-pkg.yml
```

Comment:
I think we should add a warning message here as we did in other policies and [template](https://github.com/kubewarden/go-policy-template/blob/a397b01cccaf7cfc3ddfe8077f290eac2e73e66e/Makefile#L16-L19)

Reply:
I coped that from the updated rust template, which is missing it. I'll fix that in both places